exporters/trace/jaeger/uploader.go

```diff
@@ -71,6 +87,20 @@ type AgentEndpointOptions struct {
 	agentClientUDPParams
 }
 
+// WithAgentHost sets a host to be used in the agent client endpoint.
```

Comment:
```suggestion
// WithAgentHost sets a host to be used in the agent client endpoint.
// This option overrides any value set for the
// OTEL_EXPORTER_JAEGER_AGENT_HOST environment variable.
// If this option is not passed, "localhost" will be used by default.
```

Similar for `WithAgentPort`